### PR TITLE
Remove skewed proto

### PR DESCRIPTION
**Pull request by vihu**

### CODE REVIEW

**Comment by evanmcc** [comment]
have we ever made one of these?  if so we need to keep it around no?

**Comment by evanmcc** [comment]
github UI strikes again

**Comment by vihu** [comment]
> have we ever made one of these? if so we need to keep it around no?

The only thing we've put on the chain is the `root_hash` and not the actual skewed tree

File: src/blockchain_state_channel_v1.proto [hunk 1]
```diff
@@ -3,7 +3,7 @@ syntax = "proto3";
 package helium;
 
 import "packet.proto";
-import "skewed.proto";
+// import "skewed.proto";
```
Comment (madninja) [inline src/blockchain_state_channel_v1.proto:6]: Don't comment it out, just remove it

Comment (vihu) [reply src/blockchain_state_channel_v1.proto]: Updated in 83147e0

**Pull request merged**